README.md

```diff
@@ -164,17 +165,17 @@ The following sections provide instructions for the following combinations:
 
 - Enable the non-free (closed source) repository by using your favorite editor
   to modify `/etc/apt/sources.list` and appending `contrib non-free` to the
-  `deb` respoitory
+  `deb` respository
```

Comment:
```suggestion
  `deb` repository
```

Reply:
Done.

Reply:
Still looks wrong

Reply:
Meh, I misread it because I had already fixed something similar.  Done for real this time!